src/main/java/bio/terra/common/CloudPlatformWrapper.java

```diff
@@ -56,11 +60,23 @@ void ensureValidRegion(CloudPlatform platform,
     public abstract List<? extends StorageResource<?, ?>>
         createStorageResourceValues(DatasetRequestModel datasetRequest);
 
+    public static Stream<? extends StorageResource<?, ?>>
```

Comment:
I think in general it's not great to return `Stream()` from a public method since a caller needs to know if the stream needs to be closed or not. Maybe the thing here is to weigh the cost of collecting the result vs the cost of returning a `Stream` object. Most likely the cost of the "unnecessary" collect is low. If the method wasn't `public` then I think it would be OK.

Looking at the way this code is used, I don't see why this needs to be a separate method; it's only called from the azure `createStorageResourceValues`